Updated FAQ page to Standards to follow

Yannael opened the pull request:
The FAQ page contains most of the conventions for publishing a Data Package, and I think it would be better to change its name to something more explicit like "Standards to follow".

I updated it with some recurrent corrections I ask when reviewing packages. These include mainly: The formatting of the README, the license, the formatting of the datapackage.json, and the addition of the contributors/maintainers fields.

Since I am a newbie here, I am not sure that all of these recommendations should be the best practices.

I see that most (if not all) packages so far do not follow strict guidelines. 

What is the policy, should we ensure best practices and a clear example of what an 'ideal' Data Package is, or first welcome any contribution that passes the validation and view test?

Any feedback welcome!

rgrp commented:
This is *fantastic* - really, really useful. I've made some minor comments. If you are happy to amend we'll then merge.

Yannael commented:
@rgrp Agreed with your comments, and changed to 'Best practice patterns', and removed numbering.

To help follow these best practice patterns, I think it is **very important** to have a few 'reference' Data Packages that we can refer to. 

These could be
* For tabular data: https://github.com/datasets/corruption-perceptions-index (which I maintain, so I can easily update it)
* For Geospatial data: https://github.com/datasets/geo-nuts-administrative-boundaries (@ThomasG77, since it is very well structured)

For both these packages:
* We have the datapackage.json/data folder/scripts folder
* The Readme follows the guideline 'intro/data/preparation/license'
* The datapackage.json follows the same structure 'name/title/version/description/repository/source/keywords/licenses/resource/bugs/homepage/contributors'

See:
* https://github.com/datasets/corruption-perceptions-index/blob/master/datapackage.json
* https://github.com/datasets/geo-nuts-administrative-boundaries/blob/master/datapackage.json

Questions about the datapackage.json:
* Are all the fields above worth mentioning for 'best practice patterns'? Should others be included?
* Looking at the discussion https://github.com/dataprotocols/dataprotocols/issues/130, it seems that the contributors field is the only one to recommend? (author is redundant no?)
* The details of the specifications are on http://dataprotocols.org/data-packages, which I added to the 'best practice patterns' page. Should we remove 'maintainers' from there (or at least not put it first) ?

rgrp commented:
- which are the fields above you refer to
- author - you may want this as well as contributors as it has a stronger connotation - this is the actual creator of the data package. However,  i agree contributors could be the default.
- re dataprotocols.org/data-packages - we need to remove maintainers from the spec i think as part of the 130 issue. I will get that in soon

Merging now and great contribution!
